zdm/survey.py

```diff
@@ -486,7 +488,53 @@ def calc_relative_sensitivity(DM_frb,DM,w,fbar,t_res,nu_res,model='Quadrature',d
     Probably should make this a 'self' function.... oh well, for the future!
     """
 
-def make_widths(s:Survey,wlogmean,wlogsigma,nbins,scale=2,thresh=0.5):
+def geometric_lognormals(lmu1,ls1,lmu2,ls2,bins=None,Nrand=10000,plot=False,Nbins=101):
+    '''
+    Numerically evaluates the resulting distribution of y=\sqrt{x1^2+x2^2},
+    where logx1~normal and logx2~normal with log-mean lmu and 
+    log-sigma ls.
+    This is typically used for two log-normals of intrinsic
+    FRB width and scattering time
+    
+    lmu1, ls1: log mean and log-sigma of the first distribution
```

Comment:
Can you specify the data type for each of the main inputs?
  Really helps.  Thx!

Reply:
done.